AAT_android/app/src/main/java/com/aat/activities/AttendancesActivity.java

```diff
@@ -0,0 +1,145 @@
+package com.aat.activities;
+
+import android.content.Intent;
+import android.os.AsyncTask;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ListView;
+import android.widget.ProgressBar;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+
+import com.aat.datastore.Group;
+import com.aat.interfaces.restlet.IGroupsAttendancesResource;
+import com.aat.util.EnpointUtil;
+import com.aat.util.EnpointsURL;
+import com.ase.aat_android.R;
+import com.aat.util.HashMapAdapter;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.restlet.data.MediaType;
+import org.restlet.resource.ClientResource;
+import org.restlet.resource.ResourceException;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class AttendancesActivity extends AppCompatActivity {
+    String userId = "5722646637445120";
+    private String url = EnpointsURL.HTTP_ADDRESS+EnpointsURL.REQUEST_GROUPS_STUDENT;
```

Comment:
What's the comment here?
EndpointsURL class should be an utility to keep all endpoints there. Instead of having the hardcode endpoint in each activity. Should be easier to maintain if endpoints change.